src/Messaging.DependencyInjection.Serialization.Json/JsonMessagingBuilderExtensions.cs

```diff
@@ -1,5 +1,6 @@
 ﻿using System;
 using Messaging.DependencyInjection;
+using Microsoft.Extensions.Options;
 using Serialization.Json;
 
 namespace Microsoft.Extensions.DependencyInjection
```

Comment:
Why not put these extensions in the Serialization.Json assembly? Do you envisage people using the messaging library without dependency injection for some reason?

Reply:
At this point the high level assemblies don't have any references to the Microsoft.* packages. The idea was that if project that doesn't use the `Microsoft.Extensions.DependencyInjection` like NRGS Play wants this library, it could bring in the `Messaging` and `Serialization` packages and write it up with Autofac directly.

If it's not worth the effort, we can drop these and add the DI extensions as you suggested. What do you think?

Reply:
This seems like the main reason Microsoft's DI mechanism was built... so libraries could support DI out of the box but other people could bolt on Autofac or Ninject or whatever if they wanted. Probably one to discuss with other teams but I'd be inclined to force them to get with the times personally ;-)

Reply:
Yeah, same story with Logging. Lets consider then adding to the same assembly. I still want to split the Serialization stuff from Messaging and publish two packages. DI and Logging could be coupled with MS.Extensions.*